### ISSUE TITLE
Survey of existing job boards - Exploratory Research

### ISSUE THREAD

**Issue by kljensen** [yale-research-job-board/.github#3]
We should look at other job boards, like [LinkedIn](https://linkedin.com)

**Comment by nicolayux** [comment]
I'm thinking if this should include both primary and secondary research?
Primary includes surveying potential customers;
Secondary includes browsing online forums/discussions to learn about pain points of the current job boards.

**Comment by dikshita-gn** [comment]
The primary survey would look into the high level technical structure and functionality of existing boards. And the secondary survey would deep dive into job boards that are similar to the requirement post the meeting with Tristan.

**Comment by nicolayux** [comment]
> The primary survey would look into the high level technical structure and functionality of existing boards. And the secondary survey would deep dive into job boards that are similar to the requirement post the meeting with Tristan.

I think there may be a misunderstanding about research types. Primary research involves collecting new data directly, while secondary research analyzes existing data. Perhaps we could confirm with the professors on the PRD stope, or review these definitions as a team to ensure we're all on the same page?

**Comment by dikshita-gn** [comment]
> > The primary survey would look into the high level technical structure and functionality of existing boards. And the secondary survey would deep dive into job boards that are similar to the requirement post the meeting with Tristan.
> 
> I think there may be a misunderstanding about research types. Primary research involves collecting new data directly, while secondary research analyzes existing data. Perhaps we could confirm with the professors on the PRD stope, or review these definitions as a team to ensure we're all on the same page?

Agreed. Will change the names to exploratory and conslusive reasearch for the time being. That would be more aligned to what we are doing?

**Comment by dikshita-gn** [comment]
> > > The primary survey would look into the high level technical structure and functionality of existing boards. And the secondary survey would deep dive into job boards that are similar to the requirement post the meeting with Tristan.
> > 
> > 
> > I think there may be a misunderstanding about research types. Primary research involves collecting new data directly, while secondary research analyzes existing data. Perhaps we could confirm with the professors on the PRD stope, or review these definitions as a team to ensure we're all on the same page?
> 
> Agreed. Will change the names to exploratory and conslusive reasearch for the time being. That would be more aligned to what we are doing?

**Comment by kljensen** [comment]
What is the acceptance criteria here? Seems like maybe
1. Links to existing job boards
2. Brief overview of features across these
??

**Comment by komalbagrodia97** [comment]
> What is the acceptance criteria here? Seems like maybe
> 
>     1. Links to existing job boards
> 
>     2. Brief overview of features across these
>        ??

I think we can keep the primary and secondary research as two separate stories, where:

- Primary research includes an understanding of the user's needs
- Secondary research includes an understanding of what the existing job boards offer

**Primary research:** We can create user profiles after a discussion with Tristan and then conduct interviews to understand their pain points/needs

**Secondary Research:** Gives us an understanding of the existing market. The acceptance criteria could be 
- The target audience of each competitor
- The value proposition of each competitor
- Features offered: Essential and add-ons

@dikshita-gn @nicolayux @kljensen If you agree, we can create a separate space for Primary and Secondary research and start on the latter for now.

**Comment by kljensen** [comment]
LGTM. let's not overdo either research. We can get a rough direction, build a little, and reevaluate

**Comment by dikshita-gn** [comment]
> LGTM. let's not overdo either research. We can get a rough direction, build a little, and reevaluate

Initial survey - will list what are the exisitng job boards 
1. Links
2. High level functions/features
3. Technlogy (if available)

**Comment by dikshita-gn** [comment]
> > LGTM. let's not overdo either research. We can get a rough direction, build a little, and reevaluate
> 
> Initial survey - will list what are the exisitng job boards
> 
> 1. Links
> 2. High level functions/features
> 3. Technlogy (if available)

https://yaleedu-my.sharepoint.com/:x:/r/personal/dikshita_gomathinayagam_yale_edu/_layouts/15/Doc.aspx?sourcedoc=%7B323A627A-5333-4E55-98B0-D54DA3B30F55%7D&file=Initial%20Survey.xlsx&action=default&mobileredirect=true

**Comment by fraxinus1098** [comment]
File has been moved here https://yaleedu-my.sharepoint.com/:f:/g/personal/ash_duong_yale_edu/EoMLSTgehENLge9vBqL7_uwB8vALuKBrHMPHpPQNdusRYQ?e=qpwmK8
